src/settings_gui.cpp

```diff
@@ -175,6 +175,32 @@ static StringID VolumeMarkFunc(int, int mark, int value)
 	return STR_GAME_OPTIONS_VOLUME_MARK;
 }
 
+static const int _min_small_font_size = 6;
+static const int _max_small_font_size = 20;
+
+static const int _min_medium_font_size = 8;
+static const int _max_medium_font_size = 22;
+
+static const int _min_large_font_size = 16;
+static const int _max_large_font_size = 30;
+
+static const int _min_monospaced_font_size = 8;
+static const int _max_monospaced_font_size = 22;
+
+static const int FONT_SLIDER_MARK_COUNT = 15; // Labeled at every even number. 8 Labeled marks and 7 unlabeled ones.
+
+static StringID FontSizeMarkFunction([[maybe_unused]] int mark_count, int mark, int value)
+{
+	// Skip the odd marks :
+	if ((mark & 1) == 1) {
```

Comment:
```suggestion
	if (mark % 2 == 1) {
```

No need to be fancy, can probably remove the comment as a result

Reply:
I'm reverting this change back to the original because of the code warning.